src/main/resources/static/js/errorMessages.js

```diff
@@ -0,0 +1,16 @@
+var frontendServiceUrl = $('#frontendServiceUrl').text();
+const errorsStore = [];
+function viewModel(data){
+    this.errorMessages = ko.observableArray(data);
+}
+function getErrors(){
+    const messageErr = JSON.parse(localStorage.getItem('errorsStore'));
+    var observableObject = $("#alerts")[0];
+    ko.cleanNode(observableObject);
+    ko.applyBindings(new viewModel(messageErr), observableObject);
```

Comment:
If there are no errormessages in the localstorage i think there will be problems here(nulls). Maybe that should be handled?